src/test/java/org/terasology/blockdetector/systems/BlockDetectorSystemTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.blockdetector.systems;
+
+import com.google.common.collect.Sets;
+import org.junit.Assert;
+import org.junit.Test;
+import org.terasology.audio.AudioManager;
+import org.terasology.blockdetector.utilities.DetectorData;
+import org.terasology.blockdetector.utilities.LinearAudioDetectorImpl;
+import org.terasology.context.Context;
+import org.terasology.logic.players.LocalPlayer;
+import org.terasology.logic.players.event.ResetCameraEvent;
+import org.terasology.math.Region3i;
+import org.terasology.math.geom.Vector3i;
+import org.terasology.moduletestingenvironment.ModuleTestingEnvironment;
+import org.terasology.registry.In;
+import org.terasology.world.WorldProvider;
+import org.terasology.world.block.BlockManager;
+
+
+
+public class BlockDetectorSystemTest extends ModuleTestingEnvironment {
+
+    BlockDetectorSystemImpl obj = new BlockDetectorSystemImpl();
```

Comment:
obj? That's a very unhelpful name. Change this to something clear, like `blockDetectorSystem`.